test/util/range.cpp

```diff
@@ -213,6 +213,21 @@ void test_pointers()
     BOOST_CHECK(bgr::at(r2, 5) == 5);
 }
 
+void test_const_pointers()
```

Comment:
I do not see how making the tests depend on the pre-build UTF would have allowed you to spot the problem. I spotted it immediately once I pulled after the merge of your PR, and ran the unit tests.

I am not trying to be negative. I am just trying to understand what is really the advantage of using the pre-build UTF (apart from compilation time).